allofplos/elements/article_elements.py

```diff
@@ -248,6 +248,12 @@ def get_contrib_info(contrib_element):
     except KeyError:
         # invalid contributor field; shouldn't count as contributor
         return None
+    
+    # get equal contribution status
+    try:
+        contrib_dict['equal_contrib'] = contrib_element.attrib['equal-contrib']
+    except KeyError:
+        contrib_dict['equal_contrib'] = 'No'
```

Comment:
I don't know if this should be `"No"`: what do you think about `None`? line 254 could then be
```python
contrib_dict['equal_contrib'] = contrib_element.attrib.get('equal-contrib', None)
```

Reply:
this proposed line is OK for me and also replace the whole try/except.